.eslintrc

```diff
@@ -38,7 +38,8 @@
                 "args": "none"
             }
         ],
-        "comma-dangle": 0,
+        "no-console": 1,                      // 1=warn
+        "comma-dangle": 0,                    // 0=off
```

Comment:
Also using "recommended" renders many of our rules redundant. We can nuke https://github.com/lenards/troposphere/blob/9347caf59d81baf61e1365018cd75a4e4ff4bfd0/.eslintrc#L30-L34

LINT.md will likely need an update